linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md

```diff
@@ -27,6 +27,7 @@ links:
 
   - '[linuxcommand.org](http://linuxcommand.org/lts0070.php){website}'
```

Comment:
the link seems unresponsive:
![image](https://user-images.githubusercontent.com/19802557/70982646-3cd24400-20af-11ea-93b9-b0df37573875.png)


```suggestion
```